pyproject.toml

```diff
@@ -15,6 +15,7 @@ dependencies = [
     "accelerate==0.22.0",
     "backoff==2.2.1",
     "beautifulsoup4==4.12.2",
+    "bitsandbytes==0.41.1",
```

Comment:
is this a needed change or was this something added to produce a commit? either way I'll approve, just curious

Reply:
bitsandbytes is needed to quantize  the model.